### DISCUSSION TITLE
Minimum memory limit allowed is 6MB

### DISCUSSION THREAD

**Discussion by boredland** [dokku/dokku#4959] [Q&A] [answered]
hi there! I tried to set a resource limit as documented:

`$ dokku resource:limit --memory 512 "$APP_NAME"`

thereafter, I can't deploy a container using from-image anymore:

`$ dokku git:from-image "$APP_NAME" "$(cat image_digest)"`

result:

```
-----> Generating build context
       Setting Dockerfile
-----> Updating git repository with specified build context
-----> Cleaning up...
-----> Building APP_NAME from Dockerfile
Step 1/5 : ... successful docker pull build ...
Successfully built a66257a1c8eb
Successfully tagged dokku/APP_NAME:latest
-----> Setting config vars
       DOKKU_DOCKERFILE_PORTS:  5000/tcp
-----> Releasing APP_NAME...
-----> Checking for predeploy task
-----> Executing predeploy task from app.json: yarn prisma:migrate:reset --force --skip-generate
=====> Start of APP_NAME predeploy task (5f72c646c) output
  ... successfully running predeploy...
  Done in 9.35s.
=====> End of APP_NAME predeploy task (5f72c646c) output
-----> Checking for release task
       No release task found, skipping
-----> Checking for first deploy postdeploy task
       No first deploy postdeploy task found, skipping
-----> App Procfile file found
=====> Processing deployment checks
-----> Deploying APP_NAME...
Error response from daemon: Minimum memory limit allowed is 6MB
```

any ideas? doesn't that work for from-image deployments?

**Comment by Cellane** [comment, marked as answer]
I’m kind of spit-balling here since I never used Dokku’s resource limits before, but if [this section of documentation](https://dokku.com/docs/deployment/schedulers/docker-local/#supported-resource-management-properties) is correct, then you might want to set the `--memory` parameter to `--memory 512m` (notice the unit suffix), same as [you’d do with Docker itself](https://docs.docker.com/config/containers/resource_constraints/#memory). But again, I might be totally wrong, sorry if that’s the case.
